middleware/access_log.go

```diff
@@ -366,3 +372,28 @@ func (w *bodyDumpResponseWriter) Flush() {
 func (w *bodyDumpResponseWriter) Hijack() (net.Conn, *bufio.ReadWriter, error) {
 	return w.ResponseWriter.(http.Hijacker).Hijack()
 }
+
+func MaskSensitiveInfo(reqBody []byte, maskedParams []string) ([]byte, error) {
```

Comment:
if this function only being used in this package
```suggestion
func maskSensitiveInfo(reqBody []byte, maskedParams []string) ([]byte, error) {
```